pkg/apis/scheduler/config/types.go

```diff
@@ -0,0 +1,24 @@
+package config
```

Comment:
So this is the internal version for `PlacementRequestBinderArgs` type. In that case the definitions are expected to live under `pkg/apis/scheduler/` directory to stay aligned with `pkg/apis/scheduler/VERSION` structure.

Reply:
done.